Detections/OfficeActivity/ExternalUserAddedRemovedInTeams.yaml

```diff
@@ -5,9 +5,9 @@ description: |
   one hour.This data is a part of Office 365 Connector in Azure Sentinel.'
 severity: Low
 requiredDataConnectors:
-- connectorId: Office365
-dataTypes:
-- OfficeActivity (Teams)
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
```

Comment:
If I am not mistaken, the dataType for OfficeActivity needs to be one of the following:
OfficeActivity (SharePoint) ; OfficeActivity (Exchange) ; OfficeActivity (Teams)
![image](https://user-images.githubusercontent.com/49263932/104806073-7824d200-57dd-11eb-9c9a-05b52b03238a.png)

Reply:
Changed backed.  Got a little over-zealous. :)